### DISCUSSION TITLE
Adjust the layer sizing, can StrokeWidth remain consistent?

### DISCUSSION THREAD

**Discussion by likaia** [hm21/pro_image_editor#274] [Q&A]
As shown in the figure below, I drew a rectangular area. After dragging and adjusting its size, its border became particularly thick. Can this behavior be controlled?

<img width="1381" alt="iShot Pro 2024-11-26 14 59 37" src="https://github.com/user-attachments/assets/f35ec82d-b676-4870-bf39-a810a97e46b2">

**Comment by hm21** [comment]
It’s not directly possible to achieve this. However, technically, you can listen to the `onUpdateLayer` event in `mainEditorCallbacks` and use it to update the `strokeWidth` of the layer dynamically.
